app/pages/editorg/contextCreator.js

```diff
@@ -0,0 +1,16 @@
+import manifest from './manifest.json';
+import { baseUrl } from '../../config';
+
+export const getContext = ({ organisation }) => ({
+  ...manifest,
+  organisationName: organisation.name,
+  odsCode: organisation.odsCode,
+  primaryRoleId: organisation.primaryRoleId,
+  address: organisation.address ? Object.keys(organisation.address).reduce((acc, key) => {
+    if (organisation.address[key]) acc.push(organisation.address[key]);
+    return acc;
+  }, []) : [],
```

Comment:
how about moving this to a little helper function just above this on this file.

Reply:
It is used in 3 different files so have made it's own file